src/components/Calendar.svelte

```diff
@@ -35,7 +35,7 @@
           src={currentEvent.image.src.src}
           srcset={currentEvent.image.src.srcSet.attribute}
           alt={currentEvent.image.alt}
-          sizes="(min-width: 1024px) 50vw, 100vw"
+          sizes="(min-width: 1360px) 622px, (min-width: 1040px) calc(44.33vw + 28px), 100vw"
```

Comment:
```suggestion
          sizes="(min-width: 1360px) 622px, (min-width: 1024px) 45vw, 100vw"
```

Make it less complex. `45vw` should be a close enough approximation.